draft-packages/form/KaizenDraft/Form/TextField/TextField.tsx

```diff
@@ -7,7 +7,14 @@ import { FieldGroup, FieldMessage, Input, InputProps, Label } from ".."
 
 import styles from "./styles.scss"
 
-export interface TextFieldProps extends InputProps {
+type OmittedInputProps =
+  | "startIconAdornment"
+  | "endIconAdornment"
+  | "ariaDescribedBy"
+  | "ariaLabel"
+  | "automationId"
+
+export interface TextFieldProps extends Omit<InputProps, OmittedInputProps> {
```

Comment:
I'd recommend looking at `TextAreaField/TextArea` and `Slider/InputRange` for our recommended way of defining form control prop interfaces. We are moving towards extending the native HTML/React props for the element. Happy to pair to explain more :)

Reply:
Just had a chat with @HeartSquared and realised you're already on top of all this with InputProps extending native props 😎